python/proton/_wrapper.py

```diff
@@ -138,6 +142,11 @@ def __repr__(self) -> str:
                                         self.__class__.__name__,
                                         id(self), addressof(self._impl))
 
+# print(f"log _wrapper line 144 {pn_py2void(Wrapper)=} ")
+# PYCTX = int(pn_py2void(Wrapper))
+# print(f"log _wrapper line 144 : type {type(Wrapper)} \n wrapper {Wrapper}")
+handle = ffi.new_handle(Wrapper)
+# PYCTX = int(Wrapper)
 
-PYCTX = int(pn_py2void(Wrapper))
-addressof = int
```

Comment:
It is better to directly compare with `==`, but if I really address as an `int`, then

```
int(ffi.cast("intptr_t", p))
```

https://groups.google.com/g/python-cffi/c/lwmb1xUqJRQ